### PR TITLE
Added gnupg (an alternate for python-gnupg)

### PR DESCRIPTION
**Pull request by mariusvniekerk**
This is an alternative gnupg wrapper for python, The pip package is gnupg but in the interests of not confusing this with gnupg2 itself i'm adding a -py suffix to the conda package name.
Presently we do not have a binary package built for gnupg and I can look into building one

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
ping @conda-forge/staged-recipes

File: recipes/gnupg-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - typestubs = pytypes.stubfile_2_converter:main
```
Comment (isuruf) [inline recipes/gnupg-py/meta.yaml]: Where is this coming from?

Comment (mariusvniekerk) [reply recipes/gnupg-py/meta.yaml]: remnant from another one i did,  removing

**Comment by mariusvniekerk** [comment]
Ping @isuruf

File: recipes/gnupg-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - psutil >=1.2.1
```
Comment (jakirkham) [inline recipes/gnupg-py/meta.yaml]: Is this needed during build time or is it merely a run time dependency? If the latter, could we please drop this from here?

File: recipes/gnupg-py/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (jakirkham) [inline recipes/gnupg-py/meta.yaml]: Thoughts on using `noarch: python` or is that still blocked for you?

File: recipes/gnupg-py/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - psutil >=1.2.1
```
Comment (CJ-Wright) [inline recipes/gnupg-py/meta.yaml]: Is there a python runtime dep?

**Comment by mariusvniekerk** [comment]
@CJ-Wright should be ready now

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**